www/dart-sass/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dart-sass
+version             1.32.8
+categories          www textproc
+platforms           darwin
+license             MIT
+maintainers         @queensferryme openmaintainer
+description         The reference implementation of Sass, written in Dart.
+long_description    ${description}
+homepage            https://sass-lang.com/dart-sass
+
+fetch.type          git
+git.url             https://github.com/sass/dart-sass.git
+git.branch          ad4a1692a8166a16fc57fc6f16fc5f0b15dba165
+
+use_configure       no
+
+depends_build       port:dart-sdk
+build.cmd           pub get && dart compile exe
+build.target        bin/sass.dart
```

Comment:
At least for me, `sass --version` doesn't provide any output. I find adding `build.args          -Dversion=${version}` helps.

Also if we do make use of `dart2native`, I recommend just defining your own build stage similar to some of the yarn ports, which also download deps at build time (e.g. [hyper](https://github.com/macports/macports-ports/blob/ddf03cd26ce1985f2c2837c649c65ad58cbecbcb/aqua/hyper/Portfile)).

Reply:
According to the latest dart official [documentation](https://dart.dev/tools/dart-compile), `dart2native` is now replaced by `dart compile`. Adding the `-Dversion=${version}` argument does resolve the problem, however.

Reply:
> dart2native is now replaced by dart compile

Didn't know that. Thanks @queensferryme.

> Adding the -Dversion=${version} argument does resolve the problem, however.

Yay! 🥳